tests/markdown_footnoteDefinition/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -3,7 +3,121 @@
 exports[`long.md 1`] = `
 [^hello]: this is a long long long long long long long long long long long long long paragraph.
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-[^hello]: this is a long long long long long long long long long long long long long paragraph.
+[^hello]:
+
+  this is a long long long long long long long long long long long long long
+  paragraph.
+
+`;
+
+exports[`multiline.md 1`] = `
+[^fn1]:
+
+    > \`\`\`rs
+    > fn main() {
+    >     println!("this is some Rust!");
+    > }
+    > \`\`\`
+
+[^fn2]: Here is a footnote which includes code.
+
+    \`\`\`rs
+    fn main() {
+        println!("this is some Rust!");
+    }
+    \`\`\`
+
+[^fn2]: Here is a footnote which includes code. Here is a footnote which includes code. Here is a footnote which includes code.
+
+    \`\`\`rs
+    fn main() {
+        println!("this is some Rust!");
+    }
+    \`\`\`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+[^fn1]:
+
+  > \`\`\`rs
+  > fn main() {
+  >     println!("this is some Rust!");
+  > }
+  > \`\`\`
+
+[^fn2]: Here is a footnote which includes code.
+
+  \`\`\`rs
+  fn main() {
+      println!("this is some Rust!");
+  }
+  \`\`\`
+
+[^fn2]:
+
+  Here is a footnote which includes code. Here is a footnote which includes
+  code. Here is a footnote which includes code.
+
+  \`\`\`rs
+  fn main() {
+      println!("this is some Rust!");
+  }
+  \`\`\`
+
+`;
+
+exports[`sibling.md 1`] = `
+[^a]: a
+[^a]: a
+[^a]: a
+[^a]: > 123
+[^a]: a
+[^a]: > 123
+[^a]: a
+[^a]: a
+[^a]: a
+
+---
+
+[^a]: a
+[^a]: a
+[^a]: a
+[^a]: > 123\\
+      > 456
+[^a]: a
+[^a]: > 123\\
+      > 456
+[^a]: a
+[^a]: a
+[^a]: a
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+[^a]: a
+[^a]: a
+[^a]: a
+[^a]: > 123
+[^a]: a
+[^a]: > 123
+[^a]: a
+[^a]: a
+[^a]: a
+
+---
+
+[^a]: a
+[^a]: a
+[^a]: a
+[^a]:
+
```

Comment:
This blank line looks odd to me.

Reply:
I personally think it'd better to have leading blank line there as it's a block-level node, what do you think?

1. leading + trailing
    ```md
    [^something]:

        ~~~
        hello world
        ~~~

        > blah blah blah blah blah blah blah blah
        > blah blah blah blah blah blah blah blah

    [^something]:

        ~~~
        hello world
        ~~~

        > blah blah blah blah blah blah blah blah
        > blah blah blah blah blah blah blah blah

    ```
    ```md
    [^something]:

        > blah blah blah blah blah blah blah blah
        > blah blah blah blah blah blah blah blah

    [^something]:

        > blah blah blah blah blah blah blah blah
        > blah blah blah blah blah blah blah blah

    ```
2. trailing
    ```md
    
    [^something]:
        ~~~
        hello world
        ~~~

        > blah blah blah blah blah blah blah blah
        > blah blah blah blah blah blah blah blah

    [^something]:
        ~~~
        hello world
        ~~~

        > blah blah blah blah blah blah blah blah
        > blah blah blah blah blah blah blah blah

    ```
    ```md
    [^something]:
        > blah blah blah blah blah blah blah blah
        > blah blah blah blah blah blah blah blah

    [^something]:
        > blah blah blah blah blah blah blah blah
        > blah blah blah blah blah blah blah blah

    ```

Reply:
Ah, ok. lgtm then.